packages/cra-template/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../scripts/tsconfig.json",
```

Comment:
needed to remove project config fallback to root `tsconfig,json` if tsconfig was missing in project.

this simplifies linting rules and makes our setup unified ( all packages have tsconfig and type-checking on CI )